gateway/src/apicast/policy/apicast/apicast.lua

```diff
@@ -104,7 +105,7 @@ end
 function _M:content(context)
   if not context[self].upstream then
     ngx.log(ngx.WARN, "Upstream server not found for this request")
-    return
+    return errors.upstream_not_found(context.service)
```

Comment:
What if the policy chain has a routing policy and an apicast policy with the `api_backend` defined?
This would ignore the `api_backend` in the apicast policy, right?

Reply:
Routing policy made the call() to the upstream in the content phase, so never reaches this. 

Regards.

Reply:
Sorry, when I looked at this I thought this was part of the Routing policy.

If I understood correctly, before this change, having a chain with apicast (no api_backend) -> routing worked, but now it's going to return a 404. Is that right?

Reply:
yes!

Reply:
Is this a breaking change? I think that before this, Porta always sent an api_backend. If this can be null only from this version, then I think it should be fine :+1:

Reply:
No, it's not a breaking change. 

@guicassolato the api_backend as far as I saw in the config is not nil, could you point to the right thing here?